infoblox_client/objects.py

```diff
@@ -10909,9 +10909,11 @@ class Member(InfobloxObject):
                'use_traffic_capture_auth_dns', 'use_traffic_capture_chr',
                'use_traffic_capture_qps', 'use_traffic_capture_rec_dns',
                'use_traffic_capture_rec_queries', 'use_trap_notifications',
-               'use_v4_vrrp', 'vip_setting', 'vpn_mtu']
+               'use_v4_vrrp', 'vip_setting', 'vpn_mtu', 'ipv4_address',
+               'ipv6_address']
     _search_for_update_fields = ['config_addr_type', 'host_name', 'platform',
-                                 'service_type_configuration']
+                                 'service_type_configuration', 'ipv4_address',
```

Comment:
You sure we should add ipv*_address to `_search_for_update_fields` list?

Reply:
Yes, According to the previous version (0.4.21) there are 3 places to add `ipv*_address`. https://github.com/infobloxopen/infoblox-client/blob/b8717300de27f210cd1898799228aea0ad032de4/infoblox_client/objects.py#L959